rcl_lifecycle/test/test_rcl_lifecycle.cpp

```diff
@@ -326,6 +332,7 @@ TEST(TestRclLifecycle, state_machine) {
   // Node is null
   ret = rcl_lifecycle_state_machine_fini(&state_machine, nullptr, &allocator);
   EXPECT_EQ(ret, RCL_RET_ERROR);
```

Comment:
I would expect to see `<< rcl_get_error_string().str` here as well.

Reply:
```EXPECT_EQ(ret, RCL_RET_ERROR) <<  rcl_get_error_string().str;```

It means rcl_get_error_string().str is output if ret !=  RCL_RET_ERROR.   
That is, ret is RCL_RET_OK. So I think we need not to add `<<rcl_get_error_string().str`.   
What do you think ?